pkg/apis/v1alpha1/condition.go

```diff
@@ -0,0 +1,22 @@
+package v1alpha1
+
+import corev1 "k8s.io/api/core/v1"
+
+type ConditionType string
+
+const (
+	Ready    ConditionType = "Ready"
+	NotReady ConditionType = "NotReady"
+)
+
+// Condition defines condition of CRD.
```

Comment:
How about `Condition defines the condition of the custom resource.`? Because I think it's a condition of CR, not CRD.

Reply:
Indeed, fixed.